Makefile

```
@@ -731,8 +731,14 @@ endif
 ifdef ARTIFACT_DIR
 export JUNIT_OUTPUT_PATH = ${ARTIFACT_DIR}/junit_report.xml
 endif
+# Resolve commit SHA for JUnit regression tracking (Prow presubmit → postsubmit → GitHub Actions)
+COMMIT_SHA ?= $(or $(PULL_PULL_SHA),$(PULL_BASE_SHA),$(GITHUB_SHA))
 e2e-test:
```

checkmake (0.3.2)

[warning] Target "e2e-test" defined multiple times (lines 714 and 736).

(uniquetargets)